src/bootstrap/doc.rs

```diff
@@ -725,6 +725,11 @@ impl Step for Rustc {
         cargo.rustdocflag("ena=https://docs.rs/ena/latest/");
 
         let mut to_open = None;
+
+        if self.crates.is_empty() {
+            self.crates = INTERNER.intern_list(vec!["rustc_driver".to_owned()]);
+        };
```

Comment:
Hmm, I'm not quite sure this is correct. It will document *only* rustc_driver, not any of its dependencies. I think this should be `crates = builder.crate_dependencies(rustc_driver)` or something like that.